lib/evm_logic.rb

```diff
@@ -280,9 +289,9 @@ def to_csv
           ac_csv_hash[csv_date] = @ac[csv_date].nil? ? nil : @ac[csv_date].round(2)
         end
         # evm values
-        csv << ["PV",pv_csv_hash.values.to_a].flatten!
-        csv << ["EV",ev_csv_hash.values.to_a].flatten!
-        csv << ["AC",ac_csv_hash.values.to_a].flatten!
+        csv << ['PV', pv_csv_hash.values.to_a].flatten!
+        csv << ['EV', ev_csv_hash.values.to_a].flatten!
+        csv << ['AC', ac_csv_hash.values.to_a].flatten!
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.